app.py

```diff
@@ -83,10 +83,10 @@ def process_image():
     create_directory(results_img_directory)
 
     model_directory = '/data/models/'
-    create_directory(model_directory)
+    #create_directory(model_directory)
```

Comment:
👍

Reply:
I approve this however here is a warning.

when running in production there is a huge pain regarding the model being hosted on a docker registry as its heavy with the model embedded.